core/validation/specific_validations/towns_fund_round_four.py

```diff
@@ -435,14 +433,6 @@ def spend_per_project(funding_df: pd.DataFrame, project_id: str) -> float:
     return funding_spent
 
 
-def get_allocated_funding():
-    return pd.read_csv(
-        get_file(DefaultConfig.AWS_S3_BUCKET_FILE_ASSETS, TF_FUNDING_ALLOCATED_FILE),
-        index_col="Index Code",
-        thousands=",",
```

Comment:
This is required in order to correctly parse the real allocation CSV

Reply:
Yeah, I reformated the csv to not use comma's for thousands so no need for thousands="," now